src/PrintOrder/ComponentToPrint/PrintOrderLines/utils.js

```diff
@@ -48,6 +48,10 @@ export const getColumns = (lines = []) => {
             label={<FormattedMessage id="ui-orders.itemDetails.publisher" />}
             value={line.sourceRecord?.publisher}
           />
+          <KeyValueInline
+            label={<FormattedMessage id="ui-orders.itemDetails.contributors" />}
+            value={line.sourceRecord.contributors?.map(({ contributor }) => contributor).join(', ')}
```

Comment:
Maybe it makes sense to wrap contributors in quotes?
I saw that personal contributors have format like `Lastname, Surname`, looks like for 3 contributors the result will be something like `Galt, John, Rearden, Henry, Taggart, Dagny`

Reply:
yep, makes sense